tests/app/test_encryption.py

```diff
@@ -1,20 +1,43 @@
+import pytest
+from itsdangerous import BadSignature
+
 from app.encryption import CryptoSigner
 
-signer = CryptoSigner()
 
+def test_sign_and_verify(notify_api):
```

Comment:
We don't want to put these tests in test Class?

Reply:
🤷 they have their own file. But, fair enough, it will be more consistent with how we want to organize our tests. 👍